### PR TITLE
chore: clean up unused addons artifacts

### LINKED ISSUE #2379: Clean up `parts/k8s/addons/` directory
- Remove artifact specs < 1.11
- investigate elb-svc.yaml

### PR DESCRIPTION
**Pull request by jackfrancis**
**Reason for Change**:

Clean up unused artifact sources under `parts/k8s/addons/`

**Issue Fixed**:

Fixes #2379

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: parts/k8s/addons/kubernetesmasteraddons-elb-svc.yaml [hunk 1]
```diff
@@ -1,34 +0,0 @@
----
```
Comment (jackfrancis) [inline parts/k8s/addons/kubernetesmasteraddons-elb-svc.yaml]: This manifest was not being referenced during any of the addons data marshalling/conversion into ARM customData.

Comment (CecileRobertMichon) [reply parts/k8s/addons/kubernetesmasteraddons-elb-svc.yaml]: for reference, we stopped using it intentionally in https://github.com/Azure/aks-engine/pull/1622

**Comment by jackfrancis** [comment]
Related to #2251

**Pull request merged**